time timestamp
float64[2] front_wheel_rotations
float64[2] rear_wheel_angles
geometry_msgs/Pose true_pose
